code/modules/SCP/SCPs/SCP-216.dm

```diff
@@ -0,0 +1,187 @@
+/obj/structure/scp_216
+	name = "SCP 216"
+	desc = "A metalic safe with munerical code lock."
```

Comment:
dude

Reply:
I wasn't paying attention to description.....

Reply:
"munerical" lmao